source/fundamentals/builders.txt

```diff
@@ -105,7 +105,19 @@ Use builders to check which documents in the collection have a
    var builder = Builders<Flower>.Filter;
    var filter = builder.AnyEq(f => f.Season, "winter");
 
-.. TODO for a complete list of expressions, see the Query page?
+You can also call the ``ElemMatch()`` method to find documents that have an
+array field that contains at least one element that matches a specified search
+criteria. The following example returns documents that contain the value
+``"Summer"`` in their ``Season`` array:
+
+.. code-block:: csharp
+   :copyable: true
+
+   var builder = Builders<Flower>.Filter;
+   var filter = builder.ElemMatch(f => f.Season, s => s == "Summer");
```

Comment:
S; Consider using a more complicated example to differentiate the functionality from the other array operators, but definitely ok to keep as is too

Reply:
I wanted to keep this pretty simple so users could use the same models for all of it, and also since I think the functionality is better differentiated on the server docs page. This is mostly intended just to show the method is available and show the syntax